k8s/policy/policy.go

```diff
@@ -6,7 +6,7 @@ import (
 	"sync"
 
 	"github.com/portworx/sched-ops/k8s/common"
-	policyv1beta1client "k8s.io/client-go/kubernetes/typed/policy/v1beta1"
+	policyclient "k8s.io/client-go/kubernetes"
```

Comment:
nit: We don't have to call it policyclient. It is a Kubernetes client with many APIs. So let's leave it as kubernetes.